testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/userdata/QASettingsActivity.kt

```diff
@@ -117,6 +121,28 @@ class QASettingsActivity : BaseActivity() {
         binding.edtMaxStorageLimit.setText("Current limit is $maxStorage Bytes")
 
         invalidateMaxStorageField()
+
+        binding.edtUniversalBridgeMessage.setText(settings.universalBridgeMessage)
+    }
+
+    private fun setupMiniAppMessageBridge(
+    ) {
+        miniAppMessageBridge = object : MiniAppMessageBridge() {
+            override fun sendJsonToHostApp(
+                jsonStr: String,
+                onSuccess: (content: String) -> Unit,
+                onError: (message: String) -> Unit
+            ) {
+                jsonStr.let {
+                    if (it.toString().isNotBlank()) {
+                        onSuccess(it)
+                        Toast.makeText(this@QASettingsActivity, it.toString(), Toast.LENGTH_LONG)
+                            .show()
```

Comment:
Do we need to put a `return` after this line? 🤔

Reply:
nice catch, thanks